sdk/keyvault/keyvault-keys/package.json

```diff
@@ -27,35 +27,35 @@
   "files": [
     "LICENSE.txt",
     "README.md",
-    "lib/",
-    "types/keyvault-keys.d.ts",
-    "dist/**/*.js",
-    "dist/**/*.js.map",
-    "dist/**/*.d.ts",
-    "dist/**/*.d.ts.map",
-    "dist-esm/**/*.js",
-    "dist-esm/**/*.js.map",
-    "dist-esm/**/*.d.ts",
-    "dist-esm/**/*.d.ts.map",
-    "src/**/*.ts",
-    "rollup.config.js",
+    "types/",
+    "dist/",
+    "dist-esm/",
+    "src/",
     "tsconfig.json"
   ],
+  "browser": {
+    "./dist/index.js": "./browser/azure-keyvault-keys.min.js",
+    "./dist-esm/src/index.js": "./dist-esm/src/index.js",
```

Comment:
I think this line is a no-op and should be removed.  There's no point to map a file to itself.